index.html

```diff
@@ -47,5 +53,234 @@ <h2>LET’S CONNECT</h2>
         </div>
       </div>
     </section>
+    <section class="main">
+      <section class="works">
+        <div class="card">
+          <div class="portfolio-snapshoot">
+            <img
+              src="assets/images/snapshoot-portfolio-1.png"
+              alt="portfolio-1"
+            />
+          </div>
+          <div class="left-block">
+            <div class="portfolio-info">
+              <p class="portfolio-title">TONIC</p>
```

Comment:
- [ ] for best SEO performance, the title of the cards (e.g., "Tonic") should use a header tag `hx` (h1..h6) so kindly fix all of cards ,  you can read more about it [here](https://www.reliablesoft.net/h1-tag/)